### ISSUE TITLE
Query about training details and training loss

### ISSUE THREAD

**Issue by ShinyueYao** [wgsxm/PartCrafter#41]
Thanks for your great work! I am trying to fine-tune TripoSG and PartCrafter using your example data (sword and scissors) with the default config(only changing the batch size from 32 to 24) on an A100 server. I noticed that the training loss for fine-tuning both TripoSG and PartCrafter hovers around 1.5 at beginning.

Since I am also currently working on a part-related project, I was wondering what the loss curve looked like during your actual training. Specifically, from what initial value down to what final value did the loss drop to achieve such impressive results? Thanks again for sharing your amazing work and looking forward to your reply!

**Comment by wgsxm** [comment]
Around 1.5 should be good. I suggest inferring the training progress from the generated results rather than relying on the loss values.
